goss-testing/scripts/iscsi_sbps_sanity.sh

```diff
@@ -128,3 +143,9 @@ echo "*********************************************"
 for s in $(dig -t srv +short _sbps-hsn._tcp."${host}" | sort -k3 | awk '{print $NF;}' | xargs); do printf '** %s **\n' "$s"; dig +short "$s"; done
 
 for s in $(dig -t srv +short _sbps-nmn._tcp."${host}" | sort -k3 | awk '{print $NF;}' | xargs); do printf '** %s **\n' "$s"; dig +short "$s"; done
+
+if [ -z "$A_REC" ]; then
```

Comment:
This is testing if the string `A_REC` is is empty, which will always evaluate to false (it's either `1` or `0`).  Instead of `A_REC`, at the top of the script set `EXIT_VAL=0`, then when an A record is missing set `EXIT_VAL=1`.  Then there is no need to check anything here, just end with `exit ${EXIT_VAL}`